bin/sdk_gen

```diff
@@ -193,7 +193,9 @@ const regen = async (release) => {
       return batch(
         [
           'yarn gen',
-          'bin/smoke typescript python kotlin',
+          'yarn fix', // Lint fix typescript
+          'pipenv run black python/looker_sdk/sdk/api40/*.py', // Lint fix python
+          './kotlin/gradlew -p kotlin spotlessApply', // Lint fix kotlin
```

Comment:
You gots to do what you gots to do. Would be nice for this to be a one liner to another script, but nice can come later.